apps/mobile/src/utils/localization/currency.ts

```diff
@@ -0,0 +1,80 @@
+import {bigNumberToString} from '../bigNumberToString'
+
+interface CurrencyInfo {
+    code: string                  // ISO-4217
+    flag: string                  // flag emoji
+    name: string                  // currency name
+    symbol: string                // local symbol
+    position: 'before' | 'after'  // local symbol position
+}
+
+export const currencies: Record<string, CurrencyInfo> = {
```

Comment:
Do we need to export this?

Reply:
Also, If we use the currency definition as in previous comment. We can rewrite the type definition like so: 

`const currencies: Record<Currency['code'], Currency>`. This way linter will allow us to use only keys that exists.

For fun purposes, let's imagine we don't use Zod (or branded types) in the project. It would be much better to omit explicit type definition and just put `as const` at the end of assignment, this would also make the linter aware of possible values so it won't allow accessing any `string` key but only those defined.